src/main/java/com/force/i18n/grammar/AbstractLanguageDeclension.java

```diff
@@ -609,10 +609,6 @@ public SimplePluralNounWithGender(LanguageDeclension declension, String name, St
             super(declension, name, pluralAlias, type, entityName, LanguageStartsWith.CONSONANT, gender, access, isStandardField, isCopiedFromDefault);
         }
 
-        @Override
-        public void makeSkinny() {
-        }
-
```

Comment:
this has been moved to super class -- [GrammaticalTerm.java](https://github.com/salesforce/grammaticus/pull/38/files?diff=unified&w=1#diff-fdaa532f473c1449f403a7bee5c58d9666990dcde583072f8db65e5f7c84de17R131-R133)